### PR TITLE
was added the backend skeleton

### PR DESCRIPTION
**Pull request by Melnyk9506**
Guys pleace confirm marge the sceleton

### CODE REVIEW

**Review by r-yevchuk — DISMISSED** [review]
Ok

File: server/.gitignore [hunk 1]
```diff
@@ -0,0 +1,31 @@
+HELP.md
```
Comment (vitaliimak) [inline server/.gitignore:1]: As we decided to use one .gitignore file in root project directory, could you merge files /server/.gitignore into /.gitignore

**Comment by VanderbiGit** [comment]
ok

**Comment by VanderbiGit** [comment]
ок

**Review by VanderbiGit — DISMISSED** [review]
ок

**Review by vitaliimak — COMMENTED** [review]
Could you add ReadMe.md file with information How to start server. You can add new file in /server/ directory or update /ReadMe.md in root directory

**Comment by r-yevchuk** [comment]
Ok

**Review by vitaliimak — APPROVED** [review]
Well done!

**Review by r-yevchuk — APPROVED** [review]

**Comment by VanderbiGit** [comment]
ok

**Comment by VanderbiGit** [comment]
vv

**Review by VanderbiGit — APPROVED** [review]
gogogogo

**Pull request merged**